pkg/cri/resource-manager/cache/cache.go

```diff
@@ -284,6 +284,16 @@ type Container interface {
 
 	// GetAffinity returns the annotated affinity expressions for this container.
 	GetAffinity() []*Affinity
+
+	// SetRDTClass assigns this container to the given RDT class.
+	SetRDTClass(string)
+	// GetRDTClass returns the RDT class for this container.
+	GetRDTClass() string
```

Comment:
Just a thought: could this be a generic `SetResourceClass(resType string, class string)` or similar? Would make extensions in the future easier (with `resType` prolly a type of its own instead of a string).

Reply:
True, these could be combined and/or generalized. We can do it if we start getting more of these.